src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
+
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
+}
 
+function roundNumber(area: number): number {
+  return Math.floor(area) / 100;
 }
 
-export class Triangle {
+function checkSideLength(...sides: number[]): void {
+  if (sides.some((side) => side <= 0)) {
+    throw new Error('Side can not be less/equal to zero');
+  }
+}
+
+export class Triangle implements Figure {
+  shape = Shape.triangle;
```

Comment:
Shape prop can be protected from changing
```suggestion
  readonly shape = Shape.triangle;
```